### PR TITLE
[AAE-2911] Fix decision table and script validation

### PR DESCRIPTION
**Pull request by pmartinezga**

### CODE REVIEW

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/TaskMappingsServiceTaskImplementationValidator.java [hunk 1]
```diff
@@ -109,6 +109,11 @@ public TaskMappingsServiceTaskImplementationValidator(ConnectorModelType connect
                 .map(Optional::get);
     }
 
+    private boolean isNotConnector(Optional<String> implementationTask) {
+        return implementationTask.isEmpty() || (implementationTask.get().equals("dmn-connector.EXECUTE_TABLE") && implementationTask.get()
```
Comment (ffazzini) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/TaskMappingsServiceTaskImplementationValidator.java]: should not be an OR in between dmn and script check? 🤔

Comment (pmartinezga) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/TaskMappingsServiceTaskImplementationValidator.java]: ah yes I thought I force pushed the commit fixing it, nice spot!

Comment (pmartinezga) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/TaskMappingsServiceTaskImplementationValidator.java]: done

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/TaskMappingsServiceTaskImplementationValidator.java [hunk 2]
```diff
@@ -173,4 +181,9 @@ private String getImplementationKey(String connectorName,
                             connectorName,
                             action.getName());
     }
+
+    private void addDmnScriptImplementations(Map<String, ConnectorModelFeature> availableConnectorActions){
```
Comment (ffazzini) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/TaskMappingsServiceTaskImplementationValidator.java]: this is private and unused right?

Comment (pmartinezga) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/TaskMappingsServiceTaskImplementationValidator.java]: Yes I must delete it. It is from the first approach that I discarded finally.

**Review by ffazzini — APPROVED** [review]

**Pull request merged**